### ISSUE TITLE
Get different when change batch size of test set

### ISSUE THREAD

**Issue by atonyo11** [hulianyuyy/CorrNet#61]
When I train with batch_size = 2, and test with batch_zise = 4 (or 1,3). I got different result on dev/test with test batch_zise = 2. I think it should be same. Can you explain it to me? Thank you in advance!

**Comment by hulianyuyy** [comment]
This is related to the characteristics of Batch Normalization during inferene. The mean and std of BN are calculated based on the batch size during inference.
